update-python-kata/README.md

```diff
@@ -0,0 +1,43 @@
+# FAQ: Update Python kata
+
+## What is the problem with old Python kata?
+
+The problem described here is caused by the fact that test suites of old kata do not conform to the form expected by Codewars code runner and produce test output in a format not conforming to the expected one. The main reason for that is that calls to assertions (`assert_equals`, `expect`, etc.) are not placed inside of an `@it` block, but in a scope of a `@describe` block or just in the top-level scope of tests.
+
+Additionally, missing imports for test framework and solution modules make some additional preprocessing necessary and potentially mess up line numbers in error messages.
+
+## Why was everything OK until now, and now the problem suddenly popped up?
+
+Because of future compatibility and maintenance. Python is one of the oldest supported languages on Codewars and it drags behind a large bag of compatibility hacks and workarounds. The test output is handled in the runner by a mechanism that is not related to any programming language, and workarounds needed to make Python tests work affect the maintainability of the runner overall. Test output for the majority of languages is well defined and structured, except a few which require a large set of workarounds, including Python.
+
+Fixing tests will allow for cleaning up the code of the runner and will make it easier to develop it, provide new features, and introduce new languages and language versions in the future.
```

Comment:
"introduce new languages" is not true. Python's maintainability doesn't affect other languages.

When I rewrote the runner, I made it so that each language and its test frameworks works like an add-on. So, the complexity of each language is isolated within itself.

Reply:
The only thing that keeping the old way clearly affects is the test framework. And the improved structure can improve the test output for the solvers.

I feel like the severity is misunderstood, so I'll clarify. From the runner's point of view, it's honestly not a significant maintenance cost. It's just mildly annoying. I created the list because it was requested, and it doesn't mean it should be prioritized.

I'd like to eventually add an option to use a different test framework, and this update _might_ make the migration easier, and it'll also allow our test framework to switch to use assertions.

Reply:
> I feel like the severity is misunderstood, so I'll clarify. From the runner's point of view, it's honestly not a significant maintenance cost. It's just mildly annoying. I created the list because it was requested, and it doesn't mean it should be prioritized.

I think it's just that after publishing the list cw member rushed to fix kata, what in turn triggered some authors, what in turn motivated me to create the faq trying to lower the potential temperature. I think we just got a couple of parties here who acted before they thought :)

I also think that general explanation would be better, and maybe addressing this particular even in such way is not necessary. Maybe let's just see how things are going, and I will try about something more generic and put in some part of docs on kata maintenance.

I am going to withdraw this PR and https://github.com/codewars/blog/issues/25 and think of something more general.